.travis.yml

```diff
@@ -67,9 +69,7 @@ matrix:
         # Run in `before_script` to prevent the selenium tests from still being run if the UI build fails.
         - yarn build
       script:
-        # Using Python 2's `-3` mode to surface DeprecationWarnings for Python 3 incompatibilities:
-        # https://docs.python.org/2/using/cmdline.html#cmdoption-3
-        - python -3 -m pytest tests/selenium/ --driver Firefox
```

Comment:
The `--driver Firefox` has been removed since it's already specified in `setup.cfg`